sandbox/modules/identity/build.gradle

```diff
@@ -22,6 +22,11 @@ dependencies {
 
   testImplementation project(path: ':modules:transport-netty4') // for http
   testImplementation project(path: ':plugins:transport-nio') // for http
+
+
+  implementation('org.apache.cxf:cxf-rt-rs-security-jose:3.4.5') {
```

Comment:
For my understanding, why is this change required?

Reply:
This is required in order to use the JWT functionality that cxf has. We do not handle the encryption or anything ourselves directly.